### PR TITLE
Add Debug Console component

### PR DESCRIPTION
**Pull request by Mike-MF**
**When merged this pull request will:**
- Adds some extra buttons for use on the debug console, I can think of a few nice uses for this.
- Shamelessly stolen from AF.
- Adds Mass Heal button, global event for fullHealLocal.
- Adds a make me curator button, checks if already assigned curator to prevent duping, will delete assigned modules on player disconnect.
- Adds a stop rain button.
- All buttons log to server RPT who clicked it.

### CODE REVIEW

**Review by jonpas — APPROVED** [review]
Screenshots?

**Comment by Mike-MF** [comment]
Tested alongside @JoramD0 it works.

**Review by JoramD0 — APPROVED** [review]
Tested in dedicated environment with @Mike-MF and worked as designed.

File: addons/debug_console/XEH_postInit.sqf [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#include "script_component.hpp"
+
+if (isServer) then {
+    addMissionEventHandler ["HandleDisconnect", FUNC(handleDisconnect)];
+    [QGVAR(createCurator), {_this call FUNC(addCurator)}] call CBA_fnc_addEventHandler;
```
Comment (jonpas) [inline addons/debug_console/XEH_postInit.sqf:5]:
```suggestion
    [QGVAR(createCurator), FUNC(addCurator)] call CBA_fnc_addEventHandler;
```

File: addons/debug_console/XEH_postInit.sqf [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    if (isServer) exitWith {
+        INFO_1("Mass Heal was executed by '%1'",_caller);
+    };
+    [ace_player] call ACEFUNC(medical_treatment,fullHealLocal);
+}] call CBA_fnc_addEventHandler;
+
+// Limited to 6 buttons.
+[["Curator", "Assigns as Curator"], {[QGVAR(createCurator), ace_player] call CBA_fnc_serverEvent}] call FUNC(addButton);
+[["Mass Heal", "Heals all Players"], {[QGVAR(massHeal), ace_player] call CBA_fnc_globalEvent}] call FUNC(addButton);
+[["Stop Rain", "Stops Rain"], {[QGVAR(stopRain), ace_player] call CBA_fnc_serverEvent}] call FUNC(addButton);
```
Comment (jonpas) [inline addons/debug_console/XEH_postInit.sqf:26]:
```suggestion
[["Curator", "Assigns as Curator"], {
    [QGVAR(createCurator), ace_player] call CBA_fnc_serverEvent;
}] call FUNC(addButton);
[["Mass Heal", "Heals all Players"], {
    [QGVAR(massHeal), ace_player] call CBA_fnc_globalEvent;
}] call FUNC(addButton);
[["Stop Rain", "Stops Rain"], {
    [QGVAR(stopRain), ace_player] call CBA_fnc_serverEvent;
}] call FUNC(addButton);
```

**Comment by Mike-MF** [comment]
> Does assigning curator still cause a temporary freeze?
Yes, but this one is very minor in comparison to the others. I tested it on Saturday and it was not noticeable for anyone else. (I asked)

**Review by jonpas — APPROVED** [review]

**Pull request merged**